Class Presentations/reveal-presentations/SelfConf2016.html

```diff
@@ -0,0 +1,582 @@
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+
+
+
+
+
+
+
+				<section>
+					<h2>Why Did We Decide to Do This?</h2>
+				</section>
+
+				<section>
+				<h3>We were already interested in community outreach.</h3>
+				<aside class="notes"> We had been visiting city and metro area schools
+					to speak with groups of students about what we did as app developers.
+				</aside>
+				</section>
+
+				<section>
+				<h3>We saw a need.</h3>
+				<aside class="notes"> -students asked if DL had internships, answer was
+					no
+					-we could only point them to resources and hope they figured it out
+					-knew that their peers from wealthier backgrounds could access people,
+					AP courses, clubs, etc
+				</aside>
+				</section>
+
+				<section>
+				<h3>We were familiar with teaching.</h3>
+				<aside class="notes"> -former math teacher
+					-current mother, former education student
+				</aside>
+				</section>
+
+				<section>
+					<h2>Goals</h2>
+					<ul class="checklist">
+						<li>
+							<input type="checkbox" id="goal0" checked />
+							<label for="goal0">Who Are We, and What Is This Project?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal1" checked />
+							<label for="goal1">Why Did We Decide to Do This?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal2" />
+							<label for="goal2">What Resources Did We Use?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal3" />
+							<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal4" />
+							<label for="goal4">What Are Some Things We Didn't Think About?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal5" />
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+				<section>
+					<h2>What Resources Did We Use?</h2>
+				</section>
+
+				<section>
+				<h3>For learning HTML/CSS/JavaScript:</h3>
+				<img src="../presentation-images/selfconf2016/duckett_books.jpg" width="640px" height="360px">
+				<img src="../presentation-images/selfconf2016/head_first_js.png" width="292px" height="405px">
+				<img src="../presentation-images/selfconf2016/w3_schools_logo.png" width="374px" height="48px">
+				</section>
+
+				<section>
+				<h3>For presenting HTML/CSS/JavaScript:</h3>
+				<img src="../presentation-images/selfconf2016/atom_editor.png" width="400px" height="400px">
+				<img src="../presentation-images/selfconf2016/reveal_js.jpg" width="320px" height="156px">
+				<aside class="notes">Reveal's added benefit: practice using JS</aside>
+				</section>
+
+				<section>
+				<h3>For teaching the course</h3>
+				<ul>
+					<li>Mr. Lincoln, engineering teacher extraordinaire</li>
+				</ul>
+				<aside class="notes">Really was helpful to have him with the students</aside>
```

Comment:
Add: He also hand selected the students for the program. His knowledge of the students not just academically but personally allowed us to make deeper connections quickly.